src/appshell/view/preferences/generalpreferencesmodel.cpp

```diff
@@ -215,7 +215,7 @@ GeneralPreferencesModel::StartModeList GeneralPreferencesModel::allStartupModes(
 
 QStringList GeneralPreferencesModel::scorePathFilter() const
 {
-    return { qtrc("appshell/preferences", "MuseScore file") + " (*.mscz)",
+    return { qtrc("appshell/preferences", "MuseScore Studio file") + " (*.mscz)",
```

Comment:
I think @bkunda decided to keep calling it just `MuseScore File` (on the one hand because it sounds better, on the other hand because the file format is shared within the whole MuseScore ecosystem)